### ISSUE TITLE
Timbl error in SRL

### ISSUE THREAD

**Issue by lbogaardt** [cltl/vu-rm-pip3#11]
When I tested the pipeline locally using '_cat example/test.txt | run-pipeline.sh -c pipeline.yml > output.naf_', the module SRL did not output anything. I solved this by changing in the file '_scripts\bin\vua-srl.sh_' the line

> timbl -mO:I1,2,3,4 -i $mod/25Feb2015_e-mags_mags_press_newspapers.wgt -t $FEATUREVECTOR -o $TIMBLOUTPUTFILE &>/dev/null

to

> timbl -mO:I1,2,3,4 -i $mod/25Feb2015_e-mags_mags_press_newspapers.wgt -t $FEATUREVECTOR -o $TIMBLOUTPUTFILE &>/dev/null & sleep 4

Perhaps the '_timbl_' module runs too slowly on a regular laptop to produce output before the next command is run. The '_sleep_' solves this in a hacky way, but a better solution should be possible.

**Comment by lbogaardt** [comment]
I believe the 'correct' way in Bash to tell it to wait for one line to finish before moving to the next line is simply the '_wait_' command (see [StackExchange](https://stackoverflow.com/questions/14254118/waiting-for-background-processes-to-finish-before-exiting-script)). So the following change to the script should also work:

> timbl -mO:I1,2,3,4 -i $mod/25Feb2015_e-mags_mags_press_newspapers.wgt -t $FEATUREVECTOR -o $TIMBLOUTPUTFILE &>/dev/null & wait

**Comment by vincentvanhees** [comment]
Thanks, that also works for me. 

It may be good to elaborate a bit on the description of the error that is now addressed (for future users with the same problem). When I ran the pipeline with `./run-pipeline.sh < example/test.txt > output.naf` the pipeline.log file showed the following message at the bottom:
- failed: ['vua-srl-dutch-nominal-events', 'vua-framenet-classifier']

Adjusting the code like Laurens suggests above addresses this and allows me to run the pipeline without failing parts. I am working in Ubuntu, on a relatively new 64 bit Dell XPS15 laptop (2018).
